ql/ql/src/queries/style/docs/PredicateDocs.ql

```diff
@@ -0,0 +1,48 @@
+/**
+ * @name Predicate QLDoc style.
+ * @description The QLDoc for a predicate should start with "Gets" or "Holds".
+ * @kind problem
+ * @problem.severity warning
+ * @id ql/pred-doc-style
+ * @tags maintainability
+ * @precision very-high
+ */
+
+import ql
+
+string docLines(Predicate pred) {
+  result =
+    pred.getQLDoc().getContents().replaceAll("/**", "").replaceAll("*", "").splitAt("\n").trim()
+}
+
+from Predicate pred, string message
+where
+  // only considering qldocs that look like a class-doc, to avoid reporting way too much.
+  docLines(pred).matches(["A", "An", "The"] + " %") and // looks like a class doc.
```

Comment:
Should we perhaps exclude `private` predicates? Of course ideally we would want to be consistent in our usage, but in practice we're a bit more loose when it comes to predicates that are not visible externally.